### PR TITLE
Update tutorial docs

### PR DESCRIPTION
**Pull request by timleslie**

### CODE REVIEW

File: docs/tutorials/add-lists.md [hunk 1]
```diff
@@ -6,13 +6,14 @@ order: 2
 
 # Adding Lists To Your Project
 
-We've already created one list during [previous tutorial](/docs/tutorials/new-project.md).
-Now it's the time to dive deeper. Let's make ToDos object a bit more complex.
+We've already created one list during the [previous tutorial](/docs/tutorials/new-project.md).
+Now it's the time to dive deeper. Let's make out `Todo` list a bit more complex.
```
Comment (Vultraz) [inline docs/tutorials/add-lists.md]: Small typo ("out").

File: docs/tutorials/add-lists.md [hunk 2]
```diff
@@ -6,13 +6,14 @@ order: 2
...
 
 ## Creating basic list in separate file
 
-To improve maintainability of your code it is convenient to split List schemas to separate files. Then we can import it in `index.js` for registration. Create a directory
-named 'lists', with a file `Todos.js` inside of it and put the following code inside.
+To improve maintainability of your code it is convenient to split `List` schemas to separate files.
```
Comment (MadeByMike) [inline docs/tutorials/add-lists.md]:
```suggestion
To improve maintainability of your code it is convenient to split `List` schemas into separate files.
```

File: docs/tutorials/add-lists.md [hunk 3]
```diff
@@ -6,13 +6,14 @@ order: 2
...
+We can then import it in `index.js` for registration.
+Create a directory named 'lists', with a file `Todos.js` inside of it and put the following code inside.
```
Comment (MadeByMike) [inline docs/tutorials/add-lists.md]:
```suggestion
Create a directory named 'lists', with a file `Todos.js` and put the following code inside.
```
Double inside was confusing...

File: docs/tutorials/add-lists.md [hunk 4]
```diff
@@ -6,13 +6,14 @@ order: 2
...
-Now it's the time to dive deeper. Let's make ToDos object a bit more complex.
+We've already created one list during the [previous tutorial](/docs/tutorials/new-project.md).
+Now it's the time to dive deeper. Let's make out `Todo` list a bit more complex.
 
 ## Creating basic list in separate file
 
-To improve maintainability of your code it is convenient to split List schemas to separate files. Then we can import it in `index.js` for registration. Create a directory
-named 'lists', with a file `Todos.js` inside of it and put the following code inside.
+To improve maintainability of your code it is convenient to split `List` schemas to separate files.
+We can then import it in `index.js` for registration.
```
Comment (MadeByMike) [inline docs/tutorials/add-lists.md]:
```suggestion

```
This sentences is no needed and confusing

File: docs/tutorials/add-lists.md [hunk 5]
```diff
@@ -31,7 +32,7 @@ module.exports = {
 };
 ```
 
-Here we described a very basic schema of generic ToDo. Let's register it.
+Here we described a very basic schema for a generic `Todo`. Let's add it to our Keystone system.
```
Comment (MadeByMike) [inline docs/tutorials/add-lists.md]:
```suggestion
Here we described a very basic schema for a generic `Todo`. Let's add it to our Keystone application.
```

Is application better than system (don't mind if you have reasons to prefer system)

File: docs/tutorials/add-lists.md [hunk 6]
```diff
@@ -103,7 +105,9 @@ const UsersSchema = require('./lists/Users.js');
 keystone.createList('User', UsersSchema);
 ```
 
-Relaunch your app and check if new list appeared in admin panel. Note, now `type: Password` looks when you're creating new user.
+<!-- FIXME:TL We haven't shown then how to get an Admin UI yes!!!! -->
+
+Relaunch your app and check if new the list appeared in the Admin UI. Note, now `type: Password` looks when you're creating new user.
```
Comment (MadeByMike) [inline docs/tutorials/add-lists.md]:
```suggestion
Relaunch your app and check if new the list appeared in the Admin UI. 
```

File: docs/tutorials/initial-data.md [hunk 7]
```diff
@@ -285,13 +285,13 @@ const keystone = new Keystone({
 _NOTE: When using the Array Relation syntax, If any of the queries do not match
 any items, an Error will be thrown._
 
-Clear the database, start the KeystoneJS application and visit the AdminUI. Take a look at both users, they each now have two posts associated with them. `John Duck` has the posts that contain `React` in the title. `Barry` has the posts that matched any of the queries in the array.
+Clear the database, start the Keystone JS application and visit the Admin UI. Take a look at both users, they each now have two posts associated with them. `John Duck` has the posts that contain `React` in the title. `Barry` has the posts that matched any of the queries in the array.
```
Comment (MadeByMike) [inline docs/tutorials/initial-data.md]:
```suggestion
Clear the database, start Keystone and visit the Admin UI. Take a look at both users, they each now have two posts associated with them. `John Duck` has the posts that contain `React` in the title. `Barry` has the posts that matched any of the queries in the array.
```

File: docs/tutorials/add-lists.md [hunk 8]
```diff
@@ -6,13 +6,13 @@ order: 2
 
 # Adding Lists To Your Project
 
-We've already created one list during [previous tutorial](/docs/tutorials/new-project.md).
-Now it's the time to dive deeper. Let's make ToDos object a bit more complex.
+We've already created one list during the [previous tutorial](/docs/tutorials/new-project.md).
+Now it's the time to dive deeper. Let's make out `Todo` list a bit more complex.
```
Comment (timleslie) [inline docs/tutorials/add-lists.md]:
```suggestion
Now it's the time to dive deeper. Let's make our `Todo` list a bit more complex.
```

**Pull request merged**